core/templates/pages/topics-and-skills-dashboard-page/modals/delete-skill-modal.component.html

```diff
@@ -4,17 +4,34 @@ <h3>
   </h3>
 </div>
 <div class="modal-body" *ngIf="showTopicsAssignments()">
-  <div *ngIf="allowSkillForDeletion">
-    <span>This skill is assigned to these following topics, If you delete the skill, it will be automatically removed from these topics.</span>
+  <div *ngIf="allowDeletionOfSkill">
+    <span>
+      This skill is assigned to these following topics, If you delete the
+      skill, it will automatically be removed from these topics.
+    </span>
     <div *ngFor="let topic of topicsAssignments">
       <span class="topic-name">{{ topic.topicName }}</span>
     </div>
   </div>
-  <div *ngIf="!allowSkillForDeletion"><span>It looks like, some topics contain this skill as their only diagnostic test skill. Please unassign from those topics first.</span></div>
+  <div *ngIf="!allowDeletionOfSkill">
+    <p>
+      This skill is used as the only diagnostic test skill in the following topics:
+    </p>
+    <p>
```

Comment:
Use bullet points (ul/li elements) instead. Perhaps add links to the corresponding topic editor pages, too, that open those in a new window.

Reply:
Added bullets (ul/li).